CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 2.27.4  (April 2024)
+
+- [Update]: Update OpenTok Android SDK and OpenTok iOS SDK to version 2.27.1.
+
+  This version fixes some crashes.
+
+  This version the `OTSubscriber captionReceived` event handler.
```

Comment:
captionReceived or captionsReceived. Also the line seems to be incomplete.

Reply:
It is `captionReceived`, like in the web client SDK. I fixed the incomplete sentence.